certificate-service/app/src/test/java/se/inera/intyg/certificateservice/infrastructure/logging/EventMessageServiceTest.java

```diff
@@ -0,0 +1,68 @@
+package se.inera.intyg.certificateservice.infrastructure.logging;
```

Comment:
Move this class to .messaging 

Too bad that IntelliJ doesn't move test classes when the implementation classes are moved.